libs/designer/src/lib/ui/CustomNodes/ScopeCardNode.tsx

```diff
@@ -269,6 +269,14 @@ const ScopeCardNode = ({ data, targetPosition = Position.Top, sourcePosition = P
     statusRun,
   ]);
 
+  const renderLoopsPager = useMemo(() => {
+    if (metadata?.runData?.status && metadata.runData.status !== 'InProgress') {
```

Comment:
please use equals here to make sure we are doing this case insensitively

Reply:
Updated to use the equals method